src/homework/java/salman/decision/structure/CheckOddEven.java

```diff
@@ -0,0 +1,16 @@
+package homework.java.salman.decision.structure;
+
+public class CheckOddEven {
+    public static void main(String[] args) {
+
+        int number =8;
+        if (number % 2 == 0){
+
+            System.out.println("Even number");
+        }
+        else {
+            System.out.println("Odd number");}
```

Comment:
Please organize the code so that it looks good to the poeple.

Reply:
> Please organize the code so that it looks good to the poeple.

Ok bhai..